modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -52,133 +86,174 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
-#### Hard Deprecation
+We've removed the `Drawer` component from Canvas Kit (for reference, see the
+[`Drawer`](https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/labs-drawer--basic) from v8). Please
+use the [`SidePanel`](/components/containers/side-panel/) in Preview instead.
 
-A hard-deprecated component or package is no longer available. You will need to follow the method
-prescribed in our Upgrade Guide to update your application. Please
-[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) to our team
-directly if you need additional help.
+> **Note:** The `SidePanel` in Main will eventually be replaced with the `SidePanel` in Preview. We
+> recommend you use the `SidePanel` in Preview until then.
 
 ---
 
-### Drawer
+### Layout and Column
+
+**PR:** [#2018](https://github.com/Workday/canvas-kit/pull/2018)
 
-<!---
-TODO: Please review this section with documenting SidePanel deprication
--->
+We've removed the `Layout` and `Column` components from Canvas Kit (for reference, see
+[`Column and Layout`](https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/components-containers-column-and-layout--12-column-layout)
+from v8). Please use [`Grid`](/components/layout/grid/) instead. While `Grid` is not a 1:1
+replacement for `Layout` and `Column`, it can be used to generate the same types of layouts and
+offers a more robust and flexible layout solution.
 
-We've [hard-deprecated](#hard-deprecation) `Drawer` and this component has been fully removed from
-Canvas Kit. Consider using the
-[Side Panel](https://workday.github.io/canvas-kit/?path=/story/preview-side-panel--basic). Be Aware
-that `SidePanel` from the Main package is planned to be replaced by the one from Preview. We suggest
-you use the Preview version of `SidePanel` until the one in Main is replaced with it.
+Please refer to our [Layout examples](/examples/layout) for examples of how to implement common
+layouts using `Grid`.
 
 ---
 
-### Layout and Column
+### Stack, HStack and VStack
 
-`Layout` and `Column` components have been [hard-deprecated](#hard-deprecation) and are no longer
-available in Canvas Kit. Please consider using
-[`Grid`](https://workday.github.io/canvas-kit/?path=/docs/components-containers-grid--basic)
-instead. While the `Grid` component is not a 1:1 replacement for `Layout`, it can be used in the
-same way and offers several benefits over the deprecated components as a more robust and flexible
-layout solution.
+**PR:** [#2012](https://github.com/Workday/canvas-kit/pull/2012)
 
-Please refer to
-[the Examples section](https://workday.github.io/canvas-kit/?path=/docs/examples-layouts--custom-column-positioning)
-for ideas on how to replace the deprecated `Layout` component with the `Grid` component. This will
-provide you with a deeper understanding of how to set custom column width or positioning using the
-`Grid` component.
+We've removed the `Stack`, `HStack`, and `VStack` components from Canvas Kit (for reference, see
```

Comment:
Should we mention anything about `shouldWrapChildren` and `Stack.Item` going away here? (cc: @alanbsmith, @josh-bagwell)

Reply:
We can address this in a follow-up PR.